process_report/tests/unit/test_chromium.py

```diff
@@ -0,0 +1,63 @@
+import os
+import tempfile
+
+import pandas as pd
+
+from process_report.invoices.pi_specific_invoice import PIInvoice
+from process_report.invoices import invoice
+from process_report.tests.base import BaseTestCaseWithTempDir
+
+
+class TestPIInvoiceExport(BaseTestCaseWithTempDir):
+    def setUp(self):
+        super().setUp()
+        self.output_dir = tempfile.mkdtemp()
```

Comment:
The [base class ](https://github.com/CCI-MOC/invoicing/blob/7d78602b7e7a99adee4f848211ad64537b111e35/process_report/tests/base.py#L9) already creates a temporary directory. Is there a reason why you are creating another one here? Also this folder is not being deleted.